internal/wazeroir/operations.go

```diff
@@ -819,27 +746,32 @@ func (l LabelID) FrameID() int {
 
 // ID returns the LabelID for this Label.
 func (l Label) ID() (id LabelID) {
-	id = LabelID(l.Kind) | LabelID(l.FrameID)<<32
+	id = NewLabelID(l.Kind, l.FrameID)
 	return
 }
 
+func NewLabelID(kind LabelKind, frameID uint32) LabelID {
```

Comment:
Always keep in mind to check if all the exported symbols really do need to be exported, and if so, make godoc for all of them.

Reply:
this is very import for any Go project, we take this seriously.

Reply:
good point. I think eventually we should add a lint check if possible 🤔